codebuild/spec/buildspec_analyzers.yml

```diff
@@ -39,76 +39,45 @@ version: 0.2
 # in the stack traces that ASAN reports.
 batch:
   build-list:
-    - identifier: clang_awslc
+    - identifier: clang_awslc_sanitizers
+      buildspec: codebuild/spec/buildspec_sanitizers.yml
```

Comment:
So is the plan to follow this up with another PR that adds a valgrind version of every existing asan build?

Reply:
Yes, that is the plan. I will follow this PR up by adding a CTest memcheck version (which uses valgrind) of every existing sanitizers build. This PR is for refactor purpose only.